### ISSUE TITLE
Reconnecting to hub or using history command shows custom commands by users

### ISSUE THREAD

**Issue by nishithshah2211** [HiT-Hi-FiT-Hai/hhfh-issues#27]
When a user reconnects to hub or uses `+history` command, custom commands used by users who are mods or below are also displayed, which should not happen.

**Comment by hjpotter92** [comment]
Ideally, yes. The current code has:

``` Lua
local sCmd, sData = sMessage:match( "%b<> [-+*/?!#](%w+)%s?(.*)|" )
```

which returns false positives for messages like `+1`, `*sorry`, `#some-or-other-hash-tag`. It needs a more complex solution than the one you suggest in the pull request.

**Comment by nishithshah2211** [comment]
Yes, I didn't think of that at all. We can maintain a table of commands and then check if `sCmd` is in the table or not and proceed accordingly.

**Comment by nishithshah2211** [comment]
Are there commands that do not start with a `+` or `!` or `/`? I was thinking we could use this:

``` lua
local sCmd, sData = sMessage:match( "%b<> [+/!](%w+)%s?(.*)|" )
```

**Comment by hjpotter92** [comment]
The character set `[-+*/?!#]` implies that I can use ANY of those characters for the commands. So, all of the following are equivalent:

```
!help
+help
/help
#help
?help
*help
-help
```
